stravalib/util/limiter.py

```diff
@@ -292,6 +323,9 @@ def _get_wait_time(
         elif self.priority == "low":
             return seconds_until_long_limit / (self.long_limit - long_usage)
 
+    # TODO: pyright returning - Argument of type "float | None" cannot be assigned to parameter "secs" of type "float" in function "sleep"
+    # Type "float | None" cannot be assigned to type "float"
+    # Type "None" cannot be assigned to type
```

Comment:
@enadeau question about typing here. for me pyright is flagging this suggesting Non can't be assigned. mypy does not pick up on this. Should we address this? I have heard that pyright is a bit noisier than mypy and i could also migrate to mypy in my editor as well.

Reply:
This warning is caused by modification you made on the _get_wait_time method

Reply:
ok great - yup i removed that other return type and it looks good now. thanks!!